SecretSanta/src/SecretSanta.Business/User.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SecretSanta.Business
+{
+    public class User
+    {
+        public int Id { get; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public IList<Gift> Gifts { get; }
+
+    public User(int id, string fname, string lname, IList<Gift> list)
```

Comment:
From the coding standards:
> DO NOT use abbreviations or contractions within identifier names.

```suggestion
    public User(int id, string firstName, string lastName, IList<Gift> gifts)
```